packages/webapp/src/components/Navigation/NavBar/slideMenu/index.jsx

```diff
@@ -1,8 +1,17 @@
 import { useState } from 'react';
-import clsx from 'clsx';
-import { ReactComponent as Logo } from '../../../../assets/images/navbar/nav-logo.svg';
+import { ReactComponent as Logo } from '../../../../assets/images/middle_logo.svg';
```

Comment:
Yeah, I don't like breaking beta haha ...  in case my changes got merged before you started or you built off my branch (which I normally do off of others so I am not blocked) I wanted to keep what was there beforehand.

Reply:
Ahhh, that's a good point!

Reply:
Others are comfortable with it so I can always change this habit if you think its better to just not lol.

Reply:
IMO it's okay to break beta, if we can not break it then great but if it's extra effort I don't think it's worth it since it's our testing environment basically